src/lib/protocols/tls.c

```diff
@@ -94,6 +94,8 @@ union ja3_info {
  */
 
 #define NDPI_MAX_TLS_REQUEST_SIZE 10000
+#define TLS_THRESHOLD 34186659 // Threshold for certificate validity
```

Comment:
@pacant: where does this value come from?
It seems wrong to me: max validity is 398 day -> 398 * 86400 = 34387200